lib/manageiq/appliance_console/logical_volume_management.rb

```diff
@@ -70,6 +70,7 @@ def format_logical_volume
     end
 
     def mount_disk
+      AwesomeSpawn.run!("umount", :params => ["-l", mount_point.to_s]) if File.file?("/proc/mounts") && File.read("/proc/mounts").include?(" #{mount_point} ")
```

Comment:
I think this would be more readable if it were a separate method.

Reply:
Done